src/js/other-websites/fill_relay_address.js

```diff
@@ -14,13 +15,12 @@ function fillInputWithAlias(emailInput, relayAlias) {
   // Set the value of the target field to the selected/generated mask
   emailInput.value = emailMask;
 
-  emailInput.dispatchEvent(
-    new InputEvent("relay-address", {
-      data: emailMask,
-    })
-  );
+
+  emailInput.dispatchEvent(new Event('input', {bubbles:true}));
```

Comment:
By default, events manually created and dispatched like this don't propagate (bubble) up the DOM tree like events normally do, so adding it back in more securely sets the `value` property of the `input` object.

Reply:
Wow, this is a great find - I'd not expected at all that manually-created events would not bubble! I expect this will solve issues at _many websites_ (#108, for one).

Question: do you know why there was a `{data: emailMask }` property before? It still appears to work, so I presume it wasn't necessary,